### ISSUE TITLE
A question about the real data the scale in experiments

### ISSUE THREAD

**Issue by Wang-Wenqing** [ali-design/GenRep#5]
Thanks for your great work. I have a little question about the real data scale in the comparative experiment in the paper. In the experiment setting, 'the real data encoders are trained on the ImageNet1000 dataset', the backbone is Resnet-50, and use the SimCLR data augmentation, the baseline result in Table 1 is 43.90. Is that because the experiment do not use all the training data in Imagenet1000? I didn't check the specific data, but only using resnet50 as the backbone to train a 1000-class classifier should not have such a low accuracy? Or I missing some important setting in the paper?

**Comment by Wang-Wenqing** [comment]
I found the answer in App. D. Sorry to bother you.
